### DISCUSSION TITLE
Plotting speeds on (your equipment) for posterity's sake: Dell R610, x5650 xeons: 1h45s per plot :-)

### DISCUSSION THREAD

**Discussion by polkadotbabe** [madMAx43v3r/chia-plotter#254] [General]
Dell R610, dual xeon X5650, 128GB ram, no ramdisk, 2000MB/s Seagate NVME on PCIe riser for tmp1 and tmp2:

Total plot creation time was 3645.78 sec -- yes, one hour. Used -r 22 out of 24 cores (this is a production DNS/procmail server) with a nice value of +1.

An hour on this old machine without a RAMDISK is hugely impressive. Was six to eight with the official chia plotter.

Plotted with the latest pull about 1.5 hours before this post, not seeing a build ver/time/stamp in chia_plot?

**Comment by trhr** [comment]
r730, dual xeon 2680v4s, 128gb RAM (110gb as ramdisk) + raid0 nvmes; ~21-25 minutes per plot. 5-6TiB written per day. a set of two consumer grade NVMEs are looking like they're going to last for a minimum of 4000 plots.

**Reply by aiken1811** [reply]
Please use minimum cpu ryzen 5950x for 31.8 minutes plot time

**Comment by ataa** [comment]
#452

**Comment by shimpa1** [comment]
I got to around (hard to say exactly) 5 plots/hour on a dual E5-2698V4 with 768GB RAM (2400), running 3 plots simultaneously with plotman plot manager. Only 1 tempdir (ramdisk) for all.
